### DISCUSSION TITLE
apply send-later to an autoreply filter

### DISCUSSION THREAD

**Discussion by stillewateren** [Extended-Thunder/send-later#716] [Q&A] [answered]
Hi there,

I would like to be able to plan an autoreply, triggered by a filter. The situation is this.
A client of mine has customers buying a loyalty card, and we want to send them an automatic reminder to renew their card. So, i have set up a filter for a reply with a template, but this message will be sent immediately. The reminder has to be sent 350 days after purchasing the card, but I cannot set up a time to send the reply, since I have to compose the email manually. 

I hope this makes sense :D

**Comment by jikamens** [comment]
Let me see if I understand what you're asking for here... Somebody sends you an email, that email triggers a Thunderbird filter (i.e., the built-in TB filter functionality), you would like the ability for that filter to generate a delayed, scheduled auto-reply rather than an immediate auto-reply. Is that right?

**Reply by stillewateren** [reply]
Yes, that is it.

**Comment by jikamens** [comment, marked as answer]
Yeah, OK, there's definitely no way to do that with Send Later right now. ;-) I'll open a feature request for it but unless a miracle occurs and I suddenly have a lot more time to work on Send Later than I expect or someone else steps up to pitch in, it's unlikely to be implemented any time soon.

**Reply by stillewateren** [reply]
Ok, thanks for replying quickly! I will search for another way to get this done.
